pkg/controlplane/types.go

```diff
@@ -151,3 +163,25 @@ type GetRunningUserAttributesOutput struct {
 	ControlPlaneOutput
 	UserAttributes
 }
+
+// GetJobsInput specifies how to get a job
+type GetJobsInput struct {
+	ControlPlaneInput
+}
+
+// GetJobsOutput specifies holds the response from get jobs
+type GetJobsOutput struct {
+	ControlPlaneOutput
+	JobAttributes
+}
+
+// ReloadAppServicesConfigInput specifies how to reload the app services configuration
+type ReloadAppServicesConfigInput struct {
+	ControlPlaneInput
+}
+
+// ReloadAppServicesConfigOutput specifies holds the response from reload app services config
+type ReloadAppServicesConfigOutput struct {
+	ControlPlaneOutput
+	JobAttributes
```

Comment:
bug?

Reply:
Nope, that is the schema that is returned

Reply:
Please rename to match - ReloadAppServicesConfigOutput -> ReloadAppServicesConfigJobOutput